### DISCUSSION TITLE
Codespace blocked by antivirus for phishing

### DISCUSSION THREAD

**Discussion by TrixxyLady** [community/community#28690] [Codespaces]
Hi everyone! When I tried to access my codespace today I got blocked by the antivirus for a phishing threat. This never happened before. Of course I could just overule it and get access anyway, but it made me worry. Has anyone encountered the same issue today? Thanks!

**Comment by jfullstackdev** [comment]
sometimes the antivirus itself updates which services it will block, including those legit programs because it might be that there is a similar behavior that is malicious

as for me, I turned off antivirus and firewall, I'm just making sure I will not click any links that I'm not very sure of, and I will not plug any USB stick to my computer, I'm doing this for more than 5 years and never had any single virus or worm, and my computer is as fast as it was in the beginning

**Reply by TrixxyLady** [reply]
Thanks!!! I'll just bypass github for now, and see how it goes. Have a nice day!!

**Reply by jfullstackdev** [reply]
see this one too, almost similar to your situation

https://github.com/community/community/discussions/28620
